cmd/microservice-ethereum-release-blocked-payout.go/main.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	ethAbi "github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/fluidity-money/fluidity-app/lib/log"
+	"github.com/fluidity-money/fluidity-app/lib/types/winners"
+	"github.com/fluidity-money/fluidity-app/lib/util"
+)
+
+const (
+    // EnvBlockedPayoutPayload to read blocked payout info from
+    EnvBlockedPayoutPayload = `FLU_ETHEREUM_BLOCKED_PAYOUT_PAYLOAD`
+
+    // EnvShouldPayout to determine if the reward should be paid out or just acknowledged
+    EnvShouldPayout = `FLU_ETHEREUM_PAYOUT`
+)
+
+const abiString = `
+[
+    {
+      "inputs": [
+        {
+          "internalType": "address",
+          "name": "user",
+          "type": "address"
+        },
+        {
+          "internalType": "uint256",
+          "name": "amount",
+          "type": "uint256"
+        },
+        {
+          "internalType": "bool",
+          "name": "payout",
+          "type": "bool"
+        },
+        {
+          "internalType": "uint256",
+          "name": "firstBlock",
+          "type": "uint256"
+        },
+        {
+          "internalType": "uint256",
+          "name": "lastBlock",
+          "type": "uint256"
+        }
+      ],
+      "name": "unblockReward",
+      "outputs": [],
+      "stateMutability": "nonpayable",
+      "type": "function"
+    }
+]
+`
+
+func main() {
+    reader := strings.NewReader(abiString)
+
+    unblockRewardAbi, err := ethAbi.JSON(reader)
+
+    if err != nil {
+        panic(err)
+    }
+
+    var (
+        payload    = util.GetEnvOrFatal(EnvBlockedPayoutPayload)
+        payout_    = util.GetEnvOrFatal(EnvShouldPayout)
+        payout     bool
+    )
+
+    switch payout_ {
```

Comment:
You can probably get away with checking if it's "true" and calling it there without any extra work